week03/yewon/my-blog/src/components/page/CommentWrite.jsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+
+const InputContainer = styled.div`
+  margin-top: 16px;
+`;
+
+const Input = styled.input`
+  width: 280px;
+  padding: 8px;
+  font-size: 14px;
+  margin-right: 8px;
+`;
+
+const Button = styled.button`
+  padding: 8px 12px;
+  background-color: #28a745;
+  color: white;
+  border: none;
+  border-radius: 4px;
+  cursor: pointer;
+
+  &:hover {
+    background-color: #218838;
+  }
+`;
+
+function CommentWrite({ onAddComment }) {
+  const [comment, setComment] = useState('');
+
+  const handleSubmit = () => {
+    if (!comment) return;
```

Comment:
지금은 스터디 프로젝트라 괜찮지만, 
코멘트가 없을 경우에만 얼리 리턴보다는
실제 서비스를 개발 하신다면, 이렇게 사용자에게 피드백 주는 방식도 좋은 것 같아요!
```suggestion
    if (!title.trim() || !content.trim()) {
    alert('제목과 내용을 모두 입력해주세요!');
    return;
  }
```